src/app/shared/disabled-directive.spec.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  Component,
+  DebugElement,
+} from '@angular/core';
+import {
+  ComponentFixture,
+  TestBed,
+} from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+
+import { DisabledDirective } from './disabled-directive';
+
+@Component({
+  template: `
+    <button [dsDisabled]="isDisabled">Test Button</button>
+  `,
+})
+class TestComponent {
+  isDisabled = false;
+}
+
+describe('DisabledDirective', () => {
+  let component: TestComponent;
+  let fixture: ComponentFixture<TestComponent>;
+  let button: DebugElement;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [DisabledDirective],
+      declarations: [TestComponent],
+    });
+    fixture = TestBed.createComponent(TestComponent);
+    component = fixture.componentInstance;
+    button = fixture.debugElement.query(By.css('button'));
+    fixture.detectChanges();
+  });
+
+  it('should bind aria-disabled to false initially', () => {
+    expect(button.nativeElement.getAttribute('aria-disabled')).toBe('false');
+    expect(button.nativeElement.classList.contains('disabled')).toBeFalse();
+  });
+
+  it('should bind aria-disabled to true and add disabled class when isDisabled is true', () => {
+    component.isDisabled = true;
+    fixture.detectChanges();
```

Comment:
Silly thing, but shouldn't we have a test here that explicitly sets `component.isDisabled = false` and verifies that works properly too? 

I see that we have a test above that verifies the *default* behavior is for everything to be enabled.  But, I don't see a test that verifies behavior where `isDisabled` is passed `false` as the input (and obviously the behavior should be the same as the default behavior).

This is a little bit nit picky, I know. But, I wanted to call out that we may want to add one.